### PR TITLE
Update feature board

### PR DESCRIPTION
**Pull request by raulchoque-eng**
fixed the feature board the scenario "copy board to team"

### CODE REVIEW

File: src/main/java/trello/api/rest/BoardAPIMethods.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        request = new RestClientAPI(Authentication.getRequestSpecification("admin"));
+    }
+
+    /**
+     * Creates a board with a nameBoard parameter.
+     *
+     * @param nameBoard is to set the name of board.
+     * @return a response with the name board.
+     */
+    public Response createBoard(final String nameBoard) {
```
Comment (MelissaRoman06) [inline src/main/java/trello/api/rest/BoardAPIMethods.java]: Please add these methods to TrelloAPIMethod class

File: src/main/java/trello/api/rest/TeamAPIMethods.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        request = new RestClientAPI(Authentication.getRequestSpecification("admin"));
+    }
+
+    /**
+     * Creates a team with a nameTeam parameter.
+     *
+     * @param nameTeam is to set the name of team.
+     * @return a response with the name team.
+     */
+    public Response createTeam(final String nameTeam) {
```
Comment (MelissaRoman06) [inline src/main/java/trello/api/rest/TeamAPIMethods.java]: Please add these methods to TrelloAPIMethod class

File: src/main/java/trello/api/rest/BoardAPIMethods.java (hunk 1 above)
Comment (raulchoque-eng) [reply src/main/java/trello/api/rest/BoardAPIMethods.java]: Thank you, this wrong was fixed.

File: src/main/java/trello/api/rest/TeamAPIMethods.java (hunk 2 above)
Comment (raulchoque-eng) [reply src/main/java/trello/api/rest/TeamAPIMethods.java]: Thank you, this wrong was fixed.

**Review by MelissaRoman06 — APPROVED** [review]

**Pull request merged**